docs/canvas/canvas-share-workpad.asciidoc

```diff
@@ -0,0 +1,34 @@
+[role="xpack"]
+[[workpad-share-options]]
+== Share your workpad
+
+When you are ready to share your workpad, create a PDF, or export your workpad.
+
+[float]
+[[create-workpad-pdf]]
+=== Create a PDF
+
+To view your workpad outside of Kibana, generate a PDF. 
+
+. If you are using a basic license, enable reporting in your `config/kibana.yml` file.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
+
+. Click *Generate PDF*. 
++
+image::images/canvas-generate-pdf.gif[]
+
+[float]
+[[export-workpad]]
+=== Export your workpad
+
+To share your workpads with another author, export your workpads as JSON files.
+
+. Go to the *Canvas workpads* page. 
+
+. Choose one of the following options:
+
+* To export a single workpad, click the *Export* icon next to the workpad you want to export.
```

Comment:
Instead of doing this from the Workpad selection modal, the user can also click on the `Share workpad` icon and click `Download as JSON`.
![Screen Shot 2019-08-19 at 10 43 03 AM](https://user-images.githubusercontent.com/1697105/63287018-2980aa80-c26e-11e9-8899-094dba18182f.png)